types/three/three-filmpass.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Camera, Scene, WebGLRenderTarget, WebGLRenderer, Material, Mesh, IUniform } from "./three-core";
+import {Pass} from "./three-effectcomposer";
+
+export class FilmPass extends Pass {
+    constructor(noiseIntensity: number, scanlinesIntensity: number, scanlinesCount: number, grayscale: boolean);
+    
+    scene: Scene;
+    camera: Camera;
+    uniforms: IUniform;
+    material: Material;
+    quad: Mesh;
+
+   render(renderer: WebGLRenderer, writeBuffer: WebGLRenderTarget, readBuffer: WebGLRenderTarget, delta: number, maskActive?: boolean): void;
```

Comment:
You can likely remove this render method declaration, as it's declared on `Pass`

Reply:
removed :thumbsup: